build.ps1

```diff
@@ -176,7 +176,7 @@ $HostArch = switch ($NativeProcessorArchName) {
   default { throw "Unsupported processor architecture" }
 }
 
-function Get-InstallDir($Arch) {
+function Get-ImageSwiftRoot($Arch) {
```

Comment:
What about `Get-ImageRoot` or `Get-SwiftImageRoot`?

Reply:
`Get-ImageRoot` would seem confusing given `$ImageRoot`. I prefixed everything with "Image" for uniformity. Would you prefer "SwiftImageRoot", "ToolchainImageRoot", "PlatformImageRoot", "RuntimeImageRoot" and "SDKImageRoot"?

Reply:
Yeah, I think that the swapped order it better.